MdeModulePkg/Library/BootMaintenanceManagerUiLib/BootMaintenanceManager.h

```diff
@@ -161,7 +161,7 @@ typedef enum _TYPE_OF_TERMINAL {
 #define COM1_DATA_RATE_VAR_OFFSET       VAR_OFFSET (COM1DataRate)
 #define COM1_STOP_BITS_VAR_OFFSET       VAR_OFFSET (COM1StopBits)
 #define COM1_PARITY_VAR_OFFSET          VAR_OFFSET (COM1Parity)
-#define COM1_TERMINAL_VAR_OFFSET        VAR_OFFSET (COM2TerminalType)
+#define COM1_TERMINAL_VAR_OFFSET        VAR_OFFSET (COM1TerminalType)

```

Comment:
Is this change supposed to go in this PR?

Reply:
Ahh I found this and decided to fix immediately... Anyway we don't use this at all... Up to you if you want me to separate it or not.